src/content/advanced/gitops/apps/add_appcr.md

```diff
@@ -0,0 +1,192 @@
+    to properly create respective ConfigMaps or Secrets.
+
+1. (optional - if adding configuration) Place ConfigMap and Secrets with values as the `configmap.yaml`
+  and `secret.enc.yaml` files respectively:
+
+    ```nohighlight
+    # Use one of the two for respective kind
+    cp ${APP_USER_VALUES} ./configmap.yaml
+    # cp ${APP_USER_VALUES} ./secret.enc.yaml
+    ```
+
+1. (optional - if encrypting secrets) Import the regular GPG public key of the Workload Cluster and encrypt the `secret.enc.yaml` file:
+
+    ```nohighlight
+    gpg --import management-clusters/${MC_NAME}/.sops.keys/.sops.${WC_NAME}.asc
+    sops --encrypt --in-place secret.enc.yaml
+    ```
+
+1. Go back to the `apps` directory:
+
+    ```nohighlight
+    cd ..
+    ```
+
+1. Edit the `kustomization.yaml` there adding all the newly created files as resources:
+
+    ```yaml
+    apiVersion: kustomize.config.k8s.io/v1beta1
+    kind: Kustomization
+    resources:
+    - ${APP_NAME}/appcr.yaml
+    - ${APP_NAME}/secret.enc.yaml
+    - ${APP_NAME}/configmap.yaml
+    ```
+
+  At this point, if you have followed [the WC configuration guide](/advanced/gitops/manage-workload-clusters), all the necessary Flux resources should already be configured.
+
+## Adding App using App Template
+
+1. First, you need to pick a directory with an App Template from the `bases/apps` dir created in the [template creation page](/advanced/gitops/apps/add_app_template/). Export the path to the directory in an env variable:
+
+    ```nohighlight
+    export APP_TEMPLATE_DIR=[YOUR_BASE_PATH]
+    ```
+
+    Make sure your `APP_NAME` variable is set to the exact same name as used for the app in the App Template you're pointing to.
+
+1. In the current directory (`management-clusters/${MC_NAME}/organizations/${ORG_NAME}/workload-clusters/${WC_NAME}/mapi/apps/${APP_NAME}`) create a new `kustomization.yaml` with the following content:
+
+    ```nohighlight
+    cat <<EOF > kustomization.yaml
+    apiVersion: kustomize.config.k8s.io/v1beta1
+    buildMetadata: [originAnnotations]
+    ## CONFIGURATION OVERRIDE BLOCK START - include only if overriding default config from the Template
+    configMapGenerator:
+      - files:
+          - values=override_config.yaml
+        name: \${cluster_name}-${APP_NAME}-user-values
+    generatorOptions:
+      disableNameSuffixHash: true
+    ## CONFIGURATION OVERRIDE BLOCK END
+    kind: Kustomization
+    patchesStrategicMerge:
+      - config_patch.yaml
+    resources:
+      - ../../../../../../../../../${APP_TEMPLATE_PATH}
+      - secret.enc.yaml ## ONLY IF INCLUDING SECRET
+    EOF
+    ```
+
+    Please note, that the block marked "configuration override block" is needed only if you override the default (from the Template) config and the `- secret.enc.yaml` only if you override the secret. If you don't override any, skip both parts in `kustomization.yaml` and also skip next three configuration points below.
```

Comment:
```suggestion
    Please note, that the block marked "configuration override block" is needed only if you override the default (from the Template) config and the `- secret.enc.yaml` only if you override the secret. In case you don't override any, skip both parts in `kustomization.yaml` and also the next three configuration points below.
```